packages/compass-e2e-tests/tests/search-indexes.test.ts

```diff
@@ -228,6 +235,8 @@ describe('Search Indexes', function () {
 
   for (const { name, connectionString } of connectionsWithSearchSupport) {
     context(`supports search indexes in ${name}`, function () {
+      // Set the mocha timeout to 6mins to accomodate the 4mins wait timeout
+      this.timeout(360_000);
```

Comment:
nit: You're hardcoding this one but have the other one as a variable at the top. Would probably make sense to have them both as vars at the top then you can explain it all in one comment.